README.md

```diff
@@ -19,57 +26,195 @@ pip install nvflare
 ```
 
 ## Quick Start
+The quick start guide means to help the user get FLARE up & running quickly without introducing any advanced concepts. For more details, refer to [Getting Started](https://nvflare.readthedocs.io/en/main/getting_started.html). 
+Since FLARE offers different modes of running the system, we only cover the simplest approaches here. This quick start guide uses the examples/hello-world/hello-numpy-sag as an example. You will find the details in the example's README.md file.
+We also assume you have worked with Python, already set up the virtual env. If you are new to this, please refer [Getting Started](https://nvflare.readthedocs.io/en/main/getting_started.html).
+
+#### Install NVFLARE
+```
+$ python3 -m pip install nvflare
+```
+Clone NVFLARE repo to get examples, switch main branch (latest stable branch)
+```
+$ git clone https://github.com/NVIDIA/NVFlare.git
+$ cd NVFlare
+$ git switch main
+```
+#### Quick start with CLI
+Create a temp directory as workspace and install requirements/dependencies:
+```
+$ mkdir -p /tmp/nvflare
+$ python3 -m pip install -r examples/hello-world/hello-numpy-sag/requirements.txt
+```
+* **Quick Start with Simulator**
```

Comment:
Why are this one and "Quick start with POC mode" bullet points? I think they should be sections with #### as well.

Reply:
Yes, let's just use "####" for both and remove "Quick start with CLI".

Btw, I think `mkdir -p /tmp/nvflare` is not required the folder is created by the simulator.